ansible/roles/tldraw-client-core/templates/tldraw-client-deployment.yml.j2

```
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tldraw-client-deployment
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: tldraw-client
+spec:
+  replicas: {{ TLDRAW_CLIENT_REPLICAS|default("1", true) }}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+  revisionHistoryLimit: 4
+  paused: false
+  selector:
+    labels:
+      app: tldraw-client
+      app.kubernetes.io/part-of: schulcloud-verbund
+      app.kubernetes.io/version: {{ SCHULCLOUD_SERVER_IMAGE_TAG }}
+      app.kubernetes.io/name: tldraw-client
+      app.kubernetes.io/component: tldraw
+      app.kubernetes.io/managed-by: ansible
+      git.branch: {{ SCHULCLOUD_SERVER_BRANCH_NAME }}
+      git.repo: {{ SCHULCLOUD_SERVER_REPO_NAME }}
+  template:
+    metadata:
+      labels:
+        labels:
+        app: tldraw-client
+        app.kubernetes.io/part-of: schulcloud-verbund
+        app.kubernetes.io/version: {{ SCHULCLOUD_SERVER_IMAGE_TAG }}
+        app.kubernetes.io/name: tldraw-client
+        app.kubernetes.io/component: tldraw
+        app.kubernetes.io/managed-by: ansible
+        git.branch: {{ SCHULCLOUD_SERVER_BRANCH_NAME }}
+        git.repo: {{ SCHULCLOUD_SERVER_REPO_NAME }}
+      annotations:
+    spec:
+      securityContext:  
+        runAsUser: 1000
+        runAsGroup: 1000
+        fsGroup: 1000
+        runAsNonRoot: true
+      containers:
+      - name: tldraw-client
+        image: {{ TLDRAW_CLIENT_IMAGE }}:{{ TLDRAW_CLIENT_IMAGE_TAG }}
+        imagePullPolicy: IfNotPresent
+        ports:
+        - name: http
+          containerPort: 3046
+          protocol: TCP
+        envFrom:
+        - configMapRef:
+            name: tldraw-client-configmap
+        volumeMounts:
+        - mountPath: /var/run
+          name: pid-dir
+        - mountPath: /var/cache/nginx
+          name: cache-dir
+        - mountPath: /etc/nginx/conf.d
+          name: conf-dir
+        resources:
+          limits:
+            cpu: {{ TLDRAW_CLIENT_CPU_LIMITS|default("1000m", true) }}
+            memory: {{ TLDRAW_CLIENT_MEMORY_LIMITS|default("256Mi", true) }}
+          requests:
+            cpu: {{ TLDRAW_CLIENT_CPU_REQUESTS|default("100m", true) }}
+            memory: {{ TLDRAW_CLIENT_MEMORY_REQUESTS|default("32Mi", true) }}
```

the memory is to even for dev.
```suggestion
            cpu: {{ TLDRAW_CLIENT_CPU_LIMITS|default("1000m", true) }}
            memory: {{ TLDRAW_CLIENT_MEMORY_LIMITS|default("2Gi", true) }}
          requests:
            cpu: {{ TLDRAW_CLIENT_CPU_REQUESTS|default("100m", true) }}
            memory: {{ TLDRAW_CLIENT_MEMORY_REQUESTS|default("256Mi", true) }}
```